### PR TITLE
Create 2023 Kubernetes CNCF GB seat election

### PR DESCRIPTION
**Pull request by mrbobbytables**
/cc @kubernetes/steering-committee 

/assign @jberkus 

/hold
For Review

### CODE REVIEW

File: elections/steering/2023-GB/voters.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+eligible_voters:
+- BenTheElder
+- cblecker
+- cpanato
+- justaugustus
+- mrbobbytables
+- palnabarun
+- tpepper
```
Comment (cpanato) [inline elections/steering/2023-GB/voters.yaml:8]: nit: new line

File: elections/steering/2023-GB/election.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+end_datetime: 2022-01-22 11:59:59
+no_winners: 1
+allow_no_opinion: False
+delete_after: True
+election_officers:
+  - mrbobbytables
+  - jberkus
+eligibility: Eligibility is limited to the [current Steering Committee Members](https://github.com/kubernetes/steering/tree/c0e6eafa131bb5918bd24653233568a44439497f#members)
+exception_description: No exception requests accepted for the GB election.
+exception_due: 2023-01-20 00:00:01
```
Comment (cpanato) [inline elections/steering/2023-GB/election.yaml:13]: nit: new line

File: elections/steering/2023-GB/OWNERS [hunk 3]
```diff
@@ -0,0 +1,3 @@
+approvers:
+  - jberkus
+  - mrbobbytables
```
Comment (cpanato) [inline elections/steering/2023-GB/OWNERS:3]: nit: new line

**Review by cpanato — APPROVED** [review]
only nits

/lgtm

File: elections/steering/2023-GB/OWNERS (hunk 3 above)
Comment (mrbobbytables) [reply elections/steering/2023-GB/OWNERS]: fixed em all 👍

**Comment by jberkus** [comment]
Seems fine.  I don't actually know if we'll trigger an error having candidate profiles with no body -- and would like to test that, so approving.

/lgtm
/approve

**Pull request merged**